### DISCUSSION TITLE
Particle Position Diagnostics in RZ

### DISCUSSION THREAD

**Discussion by connor-powers** [BLAST-WarpX/warpx#5448] [Q&A] [answered]
Hello! I'm running an RZ simulation and was trying to plot the average particle's radial position for one of my species. In the documentation it says that for 3D and RZ, particle positions in x, y, and z are output. However, when I look at my diagnostics, I only see <species_name>.particle_position_x and <species_name>.particle_position_y. From looking at the data it seems like <species_name>.particle_position_x is referring to the r coordinate of that particle, but if someone could confirm this that would be much appreciated. Though I do see diagnostics for particle momentums in x, y, and z, which contributed to me not being sure about whether the "x" in the particle positions corresponded to x or r. Thanks!

**Comment by connor-powers** [comment]
On second thought, could it have something to do with the fact that I set n_rz_azimuthal_modes=1?

**Comment by dpgrote** [comment, marked as answer]
Thanks for the issue. The documentation for the diagnostics is specific to the openPMD output format. When using the plotfiles with RZ, the `x` written out is the radius and `y` is the `z` position.
